ccd-adapter/src/test/java/uk/gov/hmcts/cmc/ccd/util/SampleData.java

```diff
@@ -59,5 +119,253 @@ private static CCDPaymentIntention getCCDPaymentIntention() {
             .build();
     }
 
+    private static CCDPaymentIntention getCCDPaymentIntentionImmediately() {
+        return CCDPaymentIntention.builder()
+            .paymentDate(LocalDate.now())
+            .paymentOption(CCDPaymentOption.INSTALMENTS)
+            .build();
+    }
+
+    private static CCDPaymentIntention getCCDPaymentIntentionPayBySetDate() {
+        return CCDPaymentIntention.builder()
+            .paymentDate(LocalDate.now().plusDays(10))
+            .paymentOption(CCDPaymentOption.INSTALMENTS)
+            .build();
+    }
+
+    private static CCDAddress getCCDAddress() {
+        return CCDAddress.builder()
+            .addressLine1("line1")
+            .addressLine2("line2")
+            .addressLine3("line3")
+            .postTown("city")
+            .postCode("postcode")
+            .build();
+    }
+
+    public static CCDDefendant getCCDDefendantIndividual() {
+        CCDAddress ccdAddress = getCCDAddress();
+        return CCDDefendant.builder()
+            .claimantProvidedType(INDIVIDUAL)
+            .claimantProvidedAddress(ccdAddress)
+            .claimantProvidedName("Individual")
+            .claimantProvidedDateOfBirth(LocalDate.of(1950, 1, 1))
+            .claimantProvidedServiceAddress(ccdAddress)
+            .representativeOrganisationAddress(ccdAddress)
+            .representativeOrganisationName("My Org")
+            .representativeOrganisationPhone("07987654321")
+            .representativeOrganisationEmail(",my@email.com")
```

Comment:
I will remove it but not because it is not clean but because most mail boxes do not even allow to include commas in the emailid . BTW this is existing code which I had to move into this file because you and @dharmendrak insisted.  Also this applies to where ever it is repeated.

Reply:
Please feel free to clean where ever we missed.